src/sentry/static/sentry/app/views/releases/detail/releaseHeader.jsx

```diff
@@ -47,12 +46,7 @@ export default class ReleaseHeader extends React.Component {
             <h3>
               {t('Release')}{' '}
               <strong>
-                <Version
-                  orgId={orgId}
-                  projectId={projectId}
-                  version={release.version}
-                  anchor={false}
-                />
+                <Version orgId={orgId} version={release.version} anchor={false} />
```

Comment:
Is `projectId` optional in `Version` ?

Reply:
It isn't even a defined prop in the Version component.